pkg/activator/stats_reporter.go

```diff
@@ -25,33 +25,20 @@ import (
 	"go.opencensus.io/tag"
 )
 
-// Measurement represents the type of the autoscaler metric to be reported
-type Measurement int
-
-const (
-	//RequestCountM is the request count when Activator proxy the request
-	RequestCountM = iota
-
-	// ResponseTimeInMsecM is the response time in millisecond
-	ResponseTimeInMsecM
-)
-
 var (
-	measurements = []*stats.Float64Measure{
-		RequestCountM: stats.Float64(
-			"request_count",
-			"The number of requests that are routed to Activator",
-			stats.UnitNone),
-		ResponseTimeInMsecM: stats.Float64(
-			"request_latencies",
-			"The response time in millisecond",
-			stats.UnitNone),
-	}
+	requestCountM = stats.Int64(
+		"request_count",
+		"The number of requests that are routed to Activator",
+		stats.UnitDimensionless)
+	responseTimeInMsecM = stats.Float64(
+		"request_latencies",
+		"The response time in millisecond",
+		stats.UnitDimensionless)
```

Comment:
should this be `stats.UnitMilliseconds`?

Reply:
Done.